src/app/pages/component-sidenav/_component-sidenav-theme.scss

```diff
@@ -1,36 +1,49 @@
 @import '../../../../node_modules/@angular/material/theming';
+@import '../../../styles/constants';
 
 @mixin component-viewer-sidenav-theme($theme) {
   $primary: map-get($theme, primary);
-  $accent: map-get($theme, accent);
-  $warn: map-get($theme, warn);
   $background: map-get($theme, background);
   $foreground: map-get($theme, foreground);
-  $sidenav: '.docs-component-viewer-sidenav';
+  $is-dark-theme: map-get($theme, is-dark);
+  $nav-background-opacity: if($is-dark-theme, 0.2, 0.03);
 
+  .docs-component-viewer-nav-content {
+    background: rgba(0, 0, 0, $nav-background-opacity);
```

Comment:
should this be the foreground color instead of always black?

Reply:
I wanted to make it a slight lighter accent of the current background. This is mainly done like this for different themes.